tests/test_mwan.py

```diff
@@ -295,15 +295,15 @@ def test_store_rule(e_uci, mocker):
         }
     ])
     assert mwan.store_rule(e_uci, 'rule 1', 'ns_default', 'udp', '192.168.1.1/24', '1:1024', '10.0.0.2/12',
-                           '22,443', True) == 'mwan3.ns_rule_1'
+                           '22-443', True) == 'mwan3.ns_rule_1'
     assert e_uci.get('mwan3', 'ns_rule_1') == 'rule'
     assert e_uci.get('mwan3', 'ns_rule_1', 'label') == 'rule 1'
     assert e_uci.get('mwan3', 'ns_rule_1', 'use_policy') == 'ns_default'
     assert e_uci.get('mwan3', 'ns_rule_1', 'proto') == 'udp'
     assert e_uci.get('mwan3', 'ns_rule_1', 'src_ip') == '192.168.1.1/24'
     assert e_uci.get('mwan3', 'ns_rule_1', 'src_port') == '1:1024'
     assert e_uci.get('mwan3', 'ns_rule_1', 'dest_ip') == '10.0.0.2/12'
-    assert e_uci.get('mwan3', 'ns_rule_1', 'dest_port') == '22,443'
```

Comment:
Commas are still expected to be inserted in the UCI db, since they're list of separated ports

Reply:
should be good tommaso